bootstraptest/test_yjit_new_backend.rb

```diff
@@ -303,32 +328,20 @@ def bar
     [Foo.new.foo, bar]
 }
 
+# BOP redefinition works on Integer#<
+assert_equal 'false', %q{
+  def less_than x
```

Comment:
It's a trivial topic, but I wonder whether we want to use 2 spaces or 4 spaces for indentation. From this PR, it seems like you are okay with both. If you're fine, I slightly prefer sticking to the same indentation as other bootstrap test files (2 spaces).